ambry-cloud/src/test/java/com/github/ambry/cloud/HelixVcrClusterParticipantTest.java

```diff
@@ -35,13 +35,15 @@
 import org.junit.AfterClass;
 import org.junit.Assert;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.mockito.Mockito;
 
 
 /**
  * Tests of HixVcrClusterParticipant.
  */
+@Ignore
```

Comment:
Does this change and similar ones in this PR have anything to do with moving OperationController into its own file?